src/view/com/profile/ProfileMenu.tsx

```diff
@@ -167,6 +169,14 @@ let ProfileMenu = ({
     reportDialogControl.open()
   }, [reportDialogControl])
 
+  const onPressShareATUri = React.useCallback(() => {
+    shareUrl(`at://${profile.did}`)
+  }, [profile.did])
+
+  const onPressShareDID = React.useCallback(() => {
+    shareUrl(profile.did)
```

Comment:
I should probably just set the string to clipboard directly in these instances to be honest considering these aren't URLs and in some cases (like this one) not even URIs